### DISCUSSION TITLE
无法安装任何容器

### DISCUSSION THREAD

**Discussion by AkinaLTS** [RuriOSS/daijin#33] [Q&A]
当我选择容器OS时，会提示：
Error: Can not find rootfs ！
![error](https://i.328888.xyz/2023/04/15/i7PG9P.png)
怀疑是检测到的手机架构是aarch64，而镜像站所使用的架构为arm64。以下是关于archlinux的搜索结果
![search result](https://i.328888.xyz/2023/04/15/i7PhAX.png)

**Comment by ghost** [comment]
我自己的手机测试search命令倒是没问题，cpu架构判断来自tmoe不太可能有错，或许是网络问题，后续我再研究下。话说这东西现在也太拉跨了，打字都一卡一卡的，也不知道我当时咋觉得还行的，终端console那段打算直接换c重构了，太难用。。。。咕咕咕咕咕................
